tableauserverclient/server/request_options.py

```diff
@@ -62,24 +62,49 @@ def apply_query_params(self, url):
         return "{0}?{1}".format(url, '&'.join(params))
 
 
-class ImageRequestOptions(RequestOptionsBase):
+class _FilterOptionsBase(RequestOptionsBase):
```

Comment:
Seems like a reasonable refactor.

Can you give this class a docstring just so at a glance we can tell what it's used for vs other optionsbases?